posthog/settings/web.py

```diff
@@ -41,7 +41,7 @@
     "axes",
     "drf_spectacular",
 ]
-
+DEFAULT_AUTO_FIELD = "django.db.models.AutoField"
```

Comment:
posthog/settings/data_stores.py seems like a better fit given it's a database constraint.